rfcs/2021-05-17-7469-load-balancing-aggregators.md

```diff
@@ -0,0 +1,99 @@
+# RFC 7469 - 2021-05-16 - Scaling and load balancing for Vector aggregators
+
+This RFC describes the need for a tooling and environment agnostic load balancing solution to be bundled with Vector aggregator deployments.
+
+* [Scope](#scope)
+* [Motivation](#motivation)
+* [Internal Proposal](#internal-proposal)
+* [Rationale](#rationale)
+* [Prior Art](#prior-art)
+* [Drawbacks](#drawbacks)
+* [Alternatives](#alternatives)
+  * [Do Nothing](#do-nothing)
+  * [Only Client-side Load Balancing](#only-client-side-load-balancing)
+  * [Require a Service Mesh](#require-a-service-mesh)
+* [Outstanding Questions](#outstanding-questions)
+* [Plan Of Attack](#plan-of-attack)
+
+## Scope
+
+Load balancing will be a concern for any `sink` or `source` supported by Vector; some of which can use a general solution (ex: load balancing for our HTTP based `sinks`) and some of which are specific to the component (ex: Kafka `source`/`sink`). Due to the breadth of the topic, this RFC will focus on three specific cases for load balancing while also giving consideration to future adoption for other components.
```

Comment:
It seems to me that the scope here is also focused on the sink target being a Vector instance rather than, say, an Elasticsearch cluster. I think it'd be good to make that clear.